### PR TITLE
Update Swedish

### PR DESCRIPTION
**Pull request by SimonCeder**

### CODE REVIEW

File: android/assets/jsons/translations/Swedish.properties [hunk 1]
```diff
@@ -1211,7 +1211,7 @@ Sacred Waters = Heliga Vatten
 Stone Circles = Stensättningar
 
  # Requires translation!
-Cathedrals = 
+Cathedrals = Katedraler
 Follower = Anhängare
  # Requires translation!
 May buy [building] buildings for [amount] [stat] [cityFilter] =
```
Comment (Alex25820) [inline android/assets/jsons/translations/Swedish.properties:1217]:
```suggestion
May buy [building] buildings for [amount] [stat] [cityFilter] = Kan köpa [building] byggnader för [amount] [stat] [cityFilter]
```

**Review by Alex25820 — CHANGES_REQUESTED** [review]

**Comment by yairm210** [comment]
merge translations

**Comment by yairm210** [comment]
Translations branch created

**Comment by yairm210** [comment]
Translations PR created

**Pull request merged**